app/src/main/java/org/moire/opensudoku/gui/FolderListActivity.java

```diff
@@ -92,24 +94,30 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.folder_list);
-        View getMorePuzzles = findViewById(R.id.get_more_puzzles);
+        //View getMorePuzzles = findViewById(R.id.get_more_puzzles);
 
```

Comment:
I have it commented out at the moment because I did not want to complicate importing games.